### ISSUE TITLE
Maven build incomplete

### ISSUE THREAD

**Issue by Janmm14** [Konloch/bytecode-viewer#297]
When i built with maven I got this error when trying to open a jar:

```
Exception in thread "Thread-8" Exception in thread "Thread-7" java.lang.NoClassDefFoundError: Could not initialize class the.bytecode.club.bytecodeviewer.decompilers.Decompilers
        at the.bytecode.club.bytecodeviewer.gui.ClassViewer$5.doShit(ClassViewer.java:659)
        at the.bytecode.club.bytecodeviewer.gui.PaneUpdaterThread.run(PaneUpdaterThread.java:63)
java.lang.NoClassDefFoundError: org/jd/core/v1/api/printer/Printer
        at the.bytecode.club.bytecodeviewer.decompilers.Decompilers.<clinit>(Decompilers.java:14)
        at the.bytecode.club.bytecodeviewer.gui.ClassViewer$6.doShit(ClassViewer.java:932)
        at the.bytecode.club.bytecodeviewer.gui.PaneUpdaterThread.run(PaneUpdaterThread.java:63)
Caused by: java.lang.ClassNotFoundException: org.jd.core.v1.api.printer.Printer
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:636)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:182)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:519)
        ... 3 more
```

Resulting jar is also ~10MB smaller than the jar file provided here.

Result jar I got: https://ci.janmm14.de/job/public%7Eother%7EBytecodeViewerMaven/

**Comment by ThexXTURBOXx** [comment]
I also noticed this behavior and am working on a workaround to include `system` scope dependencies in the shading process

**Comment by Janmm14** [comment]
Maybe you can override this with artifactSet config?
https://maven.apache.org/plugins/maven-shade-plugin/examples/includes-excludes.html

**Comment by ThexXTURBOXx** [comment]
> Maybe you can override this with artifactSet config?

Sadly no, since `system` scope dependency overrides anything else. It basically means "Here Maven, that jar is always present in the classpath, so just assume it is there"

**Comment by Janmm14** [comment]
Then maybe you could use [maven-dependency-plugin:unpack](https://maven.apache.org/plugins/maven-dependency-plugin/usage.html#dependency:unpack), probably in compile phase and unpack into `target/classes`?

**Comment by ThexXTURBOXx** [comment]
That's actually exactly what I did. Should be fixed in #299
